### ISSUE TITLE
Problem changing market brand

### ISSUE THREAD

**Issue by Jrrgimenezz** [peatio/peatio#600]
I have some problems to change peatio brand in market section; I've changed the entire i18n files but Im definitely doing something wrong; I've also changed the app/assets/javascripts/global_components/global.js brand var, but the problem persists.
 

Do you know what vars I need to change in order to place my brand in the market section?
The website title also says Peatio Exchange when it shouldn't, I don't know what to do and im getting crazy

**Comment by scatterp** [comment]
yes you should change the file app/views/private/markets/_header.html.slim
change the word PEATIO to what ever...

**Comment by Jrrgimenezz** [comment]
Ive changed it, but it stills shows peatio; that was the reasonable way but it isn't working.

Maybe there's problematic cache files?

**Comment by scatterp** [comment]
did you 
sudo service nginx stop
sudo service nginx start
?

did you make the mistake i made and go to "shared folder" not "private" ?

**Comment by frezeq00** [comment]
bittrex opensource: https://github.com/pkdcryptos/frezeq0_bittrex-opensource
